src/index.ts

```diff
@@ -3,4 +3,4 @@ export * from "./context";
 export * from "./types";
 export * from "./logger";
 export * from "./client";
-export { QStashWorkflowError, QStashWorkflowAbort } from "./error";
+export { WorkflowError, WorkflowAbort } from "./error";
```

Comment:
Will we make a major release? If not maybe we should do something like this for consistency
```ts
export const QStashWorkflowError = WorkflowError
```

Reply:
it will be a major release.

currently it's 0.1.2, this will be 0.2.0